Cargo.toml

```diff
@@ -29,6 +28,7 @@ fnv = "1.0.3"
 lazy_static = "0.2.1"
 libc = "0.2"
 regex = "0.1"
+atomic = "0.3"
```

Comment:
seem that we must use nightly feature for this. 

https://github.com/Amanieu/atomic-rs/#usage